Kickstarter-iOS/Info.plist

```diff
@@ -55,6 +55,8 @@
 		<array>
 			<dict>
 				<key>KitInfo</key>
+				<dict/>
+				<key>KitName</key>
```

Comment:
According to the [docs](https://www.fabric.io/kits/ios/crashlytics/install) this was missing.